repo/blob/gdrive/file_id_cache.go

```diff
@@ -0,0 +1,130 @@
+package gdrive
+
+import (
+	"sync"
+
+	"github.com/kopia/kopia/repo/blob"
+)
+
+const (
+	changeLogCacheSize = 1 << 8
+)
+
+// FileIDCache is a cache for managing the association of blobID -> fileID.
+type FileIDCache struct {
```

Comment:
nit, can you make this non-exported (lowercase)?

Reply:
Done